src/cart/index.jsx

```diff
@@ -46,49 +58,65 @@ export default class Cart extends Component {
 
   render() {
     const { cart } = this.state
+
     return (
-      <div className='container'>
-        <div className='row'>
-          <div className='col-8'>
+      <React.Fragment>
+        <section className='cart'>
+          {cart.length ? (
             <div className='container'>
-              {cart
-                ? cart.map(data => {
-                    return (
-                      <CartItem
-                        {...data}
-                        key={`cart-item-${data.id}`}
-                        removeItem={this.removeItem}
-                        setQuantity={this.setQuantity}
-                      />
-                    )
-                  })
-                : ''}
-            </div>
-          </div>
-          <div className='col-4'>
-            <div className='cart-checkout bg-white'>
-              <ul className='p-0 d-md-flex justify-content-between'>
-                <li>
-                  <span>Subtotal: </span>
-                </li>
-                <li>
-                  <span className='product-price'>
+              <h2 className='cart-title'>
+                GIỎ HÀNG{' '}
+                <small>
+                  ({cart.reduce((total, item) => total + item.quantity, 0)} sản
```

Comment:
You miss the `)` here .
`({cart.reduce((total, item) => total + item.quantity, 0)}) sản`